utils/update_json.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
It's better to add a test for `utils/update_json.sh`.
Are you also planning to use `utils/update_json.sh` instead of `local/update_json.sh` in the other recipe? (which is great)

Reply:
@sw005320 Yes. After this PR, I will remove the existing bash files under `local` in each recipe.